--- ../src-base/minecraft/net/minecraft/server/management/PlayerChunkMap.java
+++ ../src-work/minecraft/net/minecraft/server/management/PlayerChunkMap.java
@@ -48,6 +48,8 @@
     private long field_111192_g;
     private boolean field_187312_l = true;
     private boolean field_187313_m = true;
+    public final List<PlayerChunkMapEntry> lightTrackingEntries = Lists.<PlayerChunkMapEntry>newArrayList();
+    public final PlayerChunkMapEntry[] neighborChunksCache = new PlayerChunkMapEntry[6];
 
     public PlayerChunkMap(WorldServer p_i1176_1_)
     {
@@ -126,6 +128,7 @@
                 playerchunkmapentry2.func_187280_d();
             }
 
+            net.minecraftforge.common.lighting.LightTrackingHooks.onChunkMapTick(this, this.field_72697_d);
             this.field_72697_d.clear();
         }
 
